metrics-operator/converter/slo_converter.md

```diff
@@ -0,0 +1,450 @@
+# SLO Converter
+
+## Table of Contents
+
+- [SLO Converter](#slo-converter)
+  - [Table of Contents](#table-of-contents)
+  - [Description](#description)
+  - [Usage](#usage)
+  - [Conversion details](#conversion-details)
+    - [Unsupported use-cases](#unsupported-use-cases)
+      - [Criteria with 3 and more rules](#criteria-with-3-and-more-rules)
+      - [Criteria with intervals without intersection](#criteria-with-intervals-without-intersection)
+    - [Supported use-cases](#supported-use-cases)
+      - [Single rule criteria](#single-rule-criteria)
+      - [Criteria combined with logical OR operator](#criteria-combined-with-logical-or-operator)
+      - [Criteria with intervals with intersection](#criteria-with-intervals-with-intersection)
+  - [Example](#example)
```

Comment:
to make it easier using tools to keep this up2date, I'd prefer having the `<!-- toc -->` comments.

Also, we can compress it 

```suggestion
<details>
<summary>Table of Contents</summary>

<!-- toc -->

- [SLO Converter](#slo-converter)
  - [Table of Contents](#table-of-contents)
  - [Description](#description)
  - [Usage](#usage)
  - [Conversion details](#conversion-details)
    - [Unsupported use-cases](#unsupported-use-cases)
      - [Criteria with 3 and more rules](#criteria-with-3-and-more-rules)
      - [Criteria with intervals without intersection](#criteria-with-intervals-without-intersection)
    - [Supported use-cases](#supported-use-cases)
      - [Single rule criteria](#single-rule-criteria)
      - [Criteria combined with logical OR operator](#criteria-combined-with-logical-or-operator)
      - [Criteria with intervals with intersection](#criteria-with-intervals-with-intersection)
  - [Example](#example)
  
<!-- tocstop -->

</details>
```